cpp/add_synonyms.cc

```diff
@@ -220,29 +218,28 @@ void ProcessRecordTranslatedSynonyms(MARC::Record * const record, const std::vec
                                      const std::vector<std::map<std::string, std::vector<std::string>>> &translation_maps,
                                      bool * modified_record)
 {
-    std::vector<std::string>::const_iterator primary;
-    std::vector<std::string>::const_iterator output(translation_tags_and_subfield_codes.begin());
-    unsigned int i(0);
+    auto output(translation_tags_and_subfield_codes.begin());
 
     for (int lang(0); lang < LANGUAGES_END; ++lang, ++output) {
 
         std::set<std::string> synonym_values;
```

Comment:
If there can be more than around 10 entries in "synonym_values" use should use an unordered_set.